etc/helm/pachyderm/templates/dex/_config.yaml

```diff
@@ -0,0 +1,31 @@
+# TODO Maybe merge this with the pachyderm-identity-config file
+{{- define "pachyderm.dex.config" }}
+connectors:
```

Comment:
I was going to suggest to remove this connector but I guess we need to have a single connector in order to make an external dex work right?

Reply:
Pachyderm creates the connectors when bootstrapping, so I don't think we need this connector in the future. But getting rid of it is blocked behind MLDM-133